src/gatsby/src/components/park/campingDetails.js

```diff
@@ -88,7 +88,7 @@ export default function CampingDetails({ data }) {
                   .filter(count => isShown(count, parkOperation))
                   .map((count, index) => (
                     <dd key={index} className="mb-0">
-                      Total {count.display.toLowerCase()}:{" "}
+                      Total {count.display.includes("RV") ? count.display : count.display.toLowerCase()}:{" "}
```

Comment:
This label is going to appear inconsistent with the rest because it will start with a capital....
```
  {
    display: "Reservable RV-accessible sites",
    countVar: "rvSitesReservable",
    isActive: false,
  },
```